src/txn/txn_timestamp.c

```diff
@@ -668,6 +680,83 @@ __wt_txn_set_timestamp(WT_SESSION_IMPL *session, const char *cfg[])
 	return (0);
 }
 
+/*
+ * __wt_txn_parse_prepare_timestamp --
+ *	Parse a request to set a transaction's prepare_timestamp.
+ */
+int
+__wt_txn_parse_prepare_timestamp(
+    WT_SESSION_IMPL *session, const char *cfg[], wt_timestamp_t *timestamp)
+{
+	WT_CONFIG_ITEM cval;
+
+	WT_RET(__wt_config_gets_def(session,
+	    cfg, "prepare_timestamp", 0, &cval));
+	if (cval.len > 0) {
```

Comment:
Here, should we check for ret == 0 also?

Reply:
WT_RET at line 693 does that.